debian/changelog

```diff
@@ -1,3 +1,44 @@
+snapcraft (2.8) xenial; urgency=medium
+
+  [ Didier Roche ]
+  * Support .git extension detection (LP: #1566153)
+
+  [ Kyle Fazzari ]
+  * Copy plugin: Follow symlinks if outside of snap. (LP: #1532515)
+  * Lifecycle: Check to see if prerequisite is already built. (#431)
+    (LP: #1537790)
+  * Clean command: Don't automatically clean reverse dependencies. (#436)
+    (LP: #1568004)
+  * Library crawling: Make ldd failure non-fatal. (#440) (LP: #1569280)
+  * Extract step state tracking into its own module. (#438) (LP: #1477904)
+  * Automatically re-strip if dirty. (#442) (LP: #1477904)
+  * Automatically re-stage if dirty. (#443) (LP: #1477904)
+  * Detect dirty build step. (#447) (LP:  #1477904)
```

Comment:
Aw man, did I screw that up? Looks like there's one too many spaces in the bug reference.